spec/policy_machine_storage_adapters/active_record_spec.rb

```diff
@@ -26,21 +26,65 @@
     let(:policy_machine_storage_adapter) { described_class.new }
 
     describe 'find_all_of_type' do
+      it 'accepts an array parameter on a column attribute' do
+        search_uuids = ['some_uuid1', 'some_uuid2']
+        policy_machine_storage_adapter.add_object(search_uuids[0], 'some_policy_machine_uuid1')
+        policy_machine_storage_adapter.add_object(search_uuids[1], 'some_policy_machine_uuid1')
+        policy_machine_storage_adapter.add_object('some_uuid3', 'some_policy_machine_uuid1')
+
+        expect(
+          policy_machine_storage_adapter.find_all_of_type_object(
+            unique_identifier: search_uuids
+          ).count
+        ).to eq(2)
+      end
+
+      context 'when case insensitive' do
+        it 'accepts an array parameter on a column attribute' do
+          colors = ['some_uuid1', 'some_uuid2']
```

Comment:
Would prefer to see `colors` differ from the uuids of the objects, for clarity. `['burnt_umber', 'mauve']`